app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/itemsList/itemsListBody/itemsListBody.jsx

```diff
@@ -106,13 +124,21 @@ export const ItemsListBody = ({
   setModalState,
   showErrorLogs,
   optionValue,
+  isShownLess,
+  isBulkOperation,
 }) => {
   const selectItem = (id) => {
-    setModalState({
-      selectedItems: selectedItems.find((item) => item.itemId === id)
-        ? selectedItems.filter((item) => item.itemId !== id)
-        : [...selectedItems, testItems.find((item) => item.itemId === id)],
-    });
+    isBulkOperation
+      ? setModalState({
+          selectedItems: selectedItems.find((item) => item.id === id)
+            ? selectedItems.filter((item) => item.id !== id)
+            : [...selectedItems, testItems.find((item) => item.id === id)],
+        })
+      : setModalState({
+          selectedItems: selectedItems.find((item) => item.itemId === id)
+            ? selectedItems.filter((item) => item.itemId !== id)
+            : [...selectedItems, testItems.find((item) => item.itemId === id)],
+        });
   };
```

Comment:
Code seems like almost identical, maybe better provide `id` to item in `optionsSection.jsx`